### PR TITLE
Localization: add translations for twig templates

### PR DESCRIPTION
**Pull request by p365labs**
Related to #126 
Implemented symfony/translations
added 5 languages [english, italian, french, spanish, german]
- Italian has been traslated by me
- For french, spanish, german I used Google Translate

### CODE REVIEW

File: src/DoctrineAuditBundle/translations/messages.de.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+property: Eigentum
+old_value: Alter Wert
+new_value: Neuer Wert
+transaction: Transaktion
+entity_per_entity: (Entität pro Entität)
+first: Zuerst
+previous: Vorherige
+last: Letzte
+next: Weiter
+curreent:  aktuell
```
Comment (DamienHarper) [inline src/DoctrineAuditBundle/translations/messages.de.yaml]: Typo here: `current`

**Comment by DamienHarper** [comment]
@p365labs thanks very much for this contribution, very appreciated!

File: src/DoctrineAuditBundle/translations/messages.de.yaml (hunk 1 above)
Comment (p365labs) [reply src/DoctrineAuditBundle/translations/messages.de.yaml]: Fixed TYPO

**Comment by p365labs** [comment]
@DamienHarper updated the translations!

File: src/DoctrineAuditBundle/translations/messages.de.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+audit:
+  header:
+    home: Home
+    audited_entities: geprüfte Unternehmen
+  tableHeaders:
```
Comment (DamienHarper) [inline src/DoctrineAuditBundle/translations/messages.de.yaml]: Using camel case here makes the final token look weird as long as other token parts are all using snake case

File: src/DoctrineAuditBundle/translations/messages.de.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    entity: Einheit
+    table: Tabelle
+    activity: Aktivität
+  auditDetails:
```
Comment (DamienHarper) [inline src/DoctrineAuditBundle/translations/messages.de.yaml]: same here

**Comment by DamienHarper** [comment]
@p365labs apart from the last change I requested, everything looks good to me and ready to be merged 👍

File: src/DoctrineAuditBundle/translations/messages.de.yaml (hunk 2 above)
Comment (p365labs) [reply src/DoctrineAuditBundle/translations/messages.de.yaml]: yep ur right :) done!

File: src/DoctrineAuditBundle/translations/messages.de.yaml (hunk 3 above)
Comment (p365labs) [reply src/DoctrineAuditBundle/translations/messages.de.yaml]: again 👍 done

**Pull request merged**